prod-devel-rcar.yaml

```diff
@@ -109,6 +109,9 @@ common_data:
     # Install some apps to test GPU/Display
     - [IMAGE_INSTALL:append, " glmark2 kmscube"]
 
+    # gles-um-compile.bb still requires python2
+    - [I_SWEAR_TO_MIGRATE_TO_PYTHON3, "yes"]
```

Comment:
I believe it should go to `ddk_source_overrides` below

Reply:
ok, will move

Reply:
+